ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/425__model_serving_ovms_on_kserve.robot

```diff
@@ -148,3 +196,15 @@ OVMS On Kserve Suite Teardown
     Remove File    openshift_ca_istio_knative.crt
     SeleniumLibrary.Close All Browsers
     RHOSi Teardown
+
+Get OpenShift Prometheus Route
+    [Documentation]    Fetches the route for the Prometheus instance of openshift-monitoring
+    ${host}=    Run    oc get route prometheus-k8s -n openshift-monitoring -o json | jq '.status.ingress[].host'
+    ${host}=    Strip String    ${host}    characters="
+    ${route}=    Catenate    SEPARATOR=    https://    ${host}
+    RETURN    ${route}
+
+Get OpenShift Prometheus Service Account Token
+    [Documentation]    Returns a token for a service account to be used with Prometheus
+    ${token}=    Run    oc create token prometheus-k8s -n openshift-monitoring --duration 10m
+    RETURN    ${token}
```

Comment:
could you move these 2 keywords to a common file like `Resources/Page/ODH/Monitoring/Monitoring.resource`?

Reply:
Done